lib/her/model/orm.rb

```diff
@@ -13,14 +13,23 @@ def new?
 
       # Return `true` if a resource is not `#new?`
       def persisted?
-        !new?
+        !new? && !destroyed?
       end
 
       # Return whether the object has been destroyed
       def destroyed?
         @destroyed == true
       end
 
+      # Support form_for with _destroy
+      def _destroy
+        @destroy ||= false
```

Comment:
This line ends up being a little weird. I am pretty sure it will come up with the right value consistently, but it will potentially do an unnecessary assignment. What about replacing it with:
`!!@destroy`